src/nbgitpuller_downloader_dropbox/dropbox_downloader.py

```diff
@@ -1,7 +1,13 @@
-from nbgitpuller_downloader_plugins_util.plugin_helper import HandleFilesHelper
-from nbgitpuller.plugin_hook_specs import hookimpl
 import asyncio
+import nest_asyncio
+from nbgitpuller.plugin_hook_specs import hookimpl
+from nbgitpuller_downloader_plugins_util.plugin_helper import HandleFilesHelper
+
 
+# this allows us to nest usage of the event_loop from asyncio
+# being used by tornado in jupyter distro
+# Ref: https://medium.com/@vyshali.enukonda/how-to-get-around-runtimeerror-this-event-loop-is-already-running-3f26f67e762e
+nest_asyncio.apply()
```

Comment:
This will be called three times now, one per plugin, instead of once in an __init__ file or similar.

You wrote in the readme about it needing to be called but it sounds like thats a note relevant to end users users almost - its a plugin developer note right though? Hmmmm...

I think this situation merits me to read up and learn the details of this, which i currently dont understand. I think this is among the most trickiest topics.

I wonder if you needed to update install_requires in setup.py/setup.cfg along with this migration.